Advisor/ViewModel/AttractionViewModel.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  AttractionViewModel.swift
+//  Advisor
+//
+//  Created by Ильгам Ахматдинов on 30.09.2021.
+//
+
+import Foundation
+import MapKit
+import CoreLocationUI
+
+class AttractionViewModel {
+    private var networkManager = NetworkManager()
+    private var attractionData = [Attraction]()
+    
+    func fetchData(completion: @escaping () -> ()) {
+        networkManager.fetchData { result in
+            switch result {
+            case .success(let attractions):
+                self.attractionData = attractions
+                completion()
+            case .failure(let error):
+                print("Error procession json data: \(error)")
```

Comment:
how about to show any info for user about issue with loading of data from internet?